### DISCUSSION TITLE
Issues with captureEditorImage: Black Line at Bottom and Image Resolution Ratio

### DISCUSSION THREAD

**Discussion by iOSyan** [hm21/pro_image_editor#124] [Q&A] [answered]
Hello,

Question 1: 
Using `_editorKey.currentState?.captureEditorImage()` to capture a screenshot results in a black line at the bottom. Here is a sample code reference: https://github.com/hm21/pro_image_editor/discussions/106#discussion-6829551  

<img width="374" alt="图片" src="https://github.com/hm21/pro_image_editor/assets/12389901/ced9f12e-3f5c-4b1b-bc4f-dc6e1068aa23">

Question 2: 
Additionally, I want to ask if `_editorKey.currentState?.captureEditorImage` can be set to adjust the image size and resolution ratio? The captured image resolution ratio doesn’t seem to be high enough. And size is too small.

**Comment by hm21** [comment, marked as answer]
### Question 1
I used your example, but in my case it didn't show a black border at the bottom. However, you can change the output format to PNG, which should fix it.
```dart
configs: ProImageEditorConfigs(
      imageGenerationConfigs: ImageGeneratioConfigs(
        outputFormat: OutputFormat.png,
      )
)
```

### Question 2
Yes, you can also do it in the generation configs like below.
```dart
configs: ProImageEditorConfigs(
    imageGenerationConfigs: ImageGeneratioConfigs(
      /// The pixel ratio of the image relative to the content.
      customPixelRatio: MediaQuery.of(context).devicePixelRatio,

      /// The maximum output size for the image. It will maintain
      /// the image's aspect ratio but will fit within the specified
      /// constraints, similar to BoxFit.contain.
      maxOutputSize: const Size(2000, 2000),
    ),
```
The final image size is generated from your rendered image size and the pixel ratio you insert. For example, if your image is 500px wide, and you set the pixel ratio to 1, the resulting width will be 500px, and if you set the pixel ratio to 4, it will be 2000px. In the example above I use the devicePixelRatio, but feel free to use any value you want. But keep in mind that the larger the output size, the longer it will take to generate. If you want to generate the final image in a few milliseconds as a thumbnail and the high-resolution image in the background, I recommend you to check out this [example](https://github.com/hm21/pro_image_editor/blob/stable/example/lib/pages/generation_configs_example.dart) where you enable the “Generate Thumbnail” option.

**Comment by iOSyan** [comment]
It's work~ THANK YOU！
